File: sdk/java/src/test/java/com/flagent/client/model/ErrorTest.java
package com.flagent.client.model;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

/**
 * Model tests for Error
 */
class ErrorTest {

    @Test
    void testError() {
        Error model = new Error().message("Something went wrong");
        Assertions.assertNotNull(model);
        Assertions.assertEquals("Something went wrong", model.getMessage());
    }

    @Test
    void messageTest() {
        Error model = new Error();
        model.setMessage("Validation failed");
        Assertions.assertEquals("Validation failed", model.getMessage());
    }

}

File: sdk/dart/test/error_test.dart
import 'package:test/test.dart';
import 'package:flagent_client/flagent_client.dart';

// tests for Error
void main() {
  final instance = Error((b) => b..message = 'Something went wrong');

  group(Error, () {
    test('to test the property `message`', () async {
      expect(instance.message, equals('Something went wrong'));
    });
  });
}
